src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -256,6 +256,19 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model, s
     public IActionResult ChangePasswordConfirmation()
         => View();
 
+    /// <summary>
+    /// This action is retained for backward compatibility.
+    /// Both this action and <see cref="ExternalAuthenticationsStartupFilter"/> are scheduled for removal in version 3.
+    /// </summary>
+    /// <returns></returns>
+    [HttpPost]
+    [AllowAnonymous]
+    [ValidateAntiForgeryToken]
+    public IActionResult ExternalLogin()
```

Comment:
What's the purpose of this action, exactly? In 2.0, it was `IActionResult ExternalLogin(string provider, string returnUrl = null)`, so I don't think routes to that one with parameters will match this one.

Reply:
Yeah, it actually matches despite the parameters not being there. Weird, but good then.